lerobot/common/robot_devices/robots/koch.py

```diff
@@ -474,21 +498,53 @@ def capture_observation(self):
             obs_dict[f"observation.images.{name}"] = torch.from_numpy(images[name])
         return obs_dict
 
-    def send_action(self, action: torch.Tensor):
-        """The provided action is expected to be a vector."""
+    def send_action(self, action: torch.Tensor, follower_names: list[str] | None = None):
```

Comment:
@aliberts @Cadene I had to make this change (adding `follower_arms` kwarg) as `teleop_step` wants to set the follower arms one by one and get a dt for each. We could have also just not gotten a dt for each arm.